docs/mimxrt/quickref.rst

```diff
@@ -537,6 +537,62 @@ port and LAN(1) for the 1G port.
 
 For details of the network interface refer to the class :ref:`network.LAN <network.LAN>`.
 
+class Encoder -- Quadrature Encoder for i.MXRT MCUs
+---------------------------------------------------
+
+This class provides the Quadrature Encoder Service.
+
+Example usage::
+
+    # Samples for Teensy
+    #
+
+    from machine import Pin, Encoder
+
+    qe = Encoder(0, Pin(0), Pin(1)) # create Quadrature Encoder object
+    qe.value()                      # get current counter values
+    qe.value(0)                     # Set value and cycles to 0
+    qe.init(cpc=128)                # Specify 128 counts/cycle
+    qe.init(index=Pin(3))           # Specify Pin 3 as Index pulse input
+    qe.deinit()                     # turn off the Quadrature Encoder
+    qe.init(match=64)               # Set a match event at count 64
```

Comment:
I flashed my teensy 4.1 with this branch's build. I am following this doc and can see values. When I attempt to use this match keyword I receive an error.

```
>>> e0.init(match=100)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: extra keyword arguments given
```

Reply:
Thank you for having a look at that class. It's always good if other people look at that. Here, the documentation has to be updated, since the match=xxx argument was removed from the init() call. That was done as result of trying to harmonize the various implementations. The same applies to the `reverse`keyword argument.